src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total;
+  const rentPrice = 40;
+
```

Comment:
4. [REUSABILITY] - Give variables abstract names - it's will be easiar for another person to read your code
5. [VARIABLES] - Don't hardcode some values that may change in the future in variable names (numbers etc.)

Name the variables of discount. Look examples in checklist.